tests/test_hra.py

```diff
@@ -1146,3 +1146,56 @@ def test_validate_negative_resolution(self):
         expected_error = (['resolution'],
                           'Value does not meet condition value > 0')
         self.assertTrue(expected_error in validation_error_list)
+
+    @staticmethod
+    def _assert_vectors_equal(a_vector_path, b_vector_path, precision=6):
+        """Assert that geometries in two vectors are equal, order-insensitive.
+
+        Args:
+            a_vector_path (str): a path to a valid OGR vector.
+            b_vector_path (str): a path to a valid OGR vector.
+
+        Returns:
+            None.
+
+        Raises:
+            AssertionError when the two point geometries are not equal up to
+                desired precision (default is 1E-6).
+
+        """
+        a_vector = gdal.OpenEx(a_vector_path, gdal.OF_VECTOR)
+        a_layer = a_vector.GetLayer(0)
+        a_feat = a_layer.GetNextFeature()
+
+        b_vector = gdal.OpenEx(b_vector_path, gdal.OF_VECTOR)
+        b_layer = b_vector.GetLayer(0)
+        b_feat = b_layer.GetNextFeature()
+
+        while a_feat is not None:
+            # Get coordinates from geometry and store them in a list
+            a_geom = a_feat.GetGeometryRef()
+            a_geom_list = re.findall(r'\d+\.\d+', a_geom.ExportToWkt())
+            a_geom_list = [float(x) for x in a_geom_list]
+
+            b_geom = b_feat.GetGeometryRef()
+            b_geom_list = re.findall(r'\d+\.\d+', b_geom.ExportToWkt())
+            b_geom_list = [float(x) for x in b_geom_list]
+
+            try:
+                numpy.testing.assert_allclose(
+                    a_geom_list, b_geom_list, rtol=0, atol=10**-precision)
+            except AssertionError:
+                a_feature_fid = a_feat.GetFID()
+                b_feature_fid = b_feat.GetFID()
+                raise AssertionError('Geometries are not equal in feature %s, '
+                                     'regression feature %s in layer 0' %
+                                     (a_feature_fid, b_feature_fid))
+            a_feat = None
+            b_feat = None
+            a_feat = a_layer.GetNextFeature()
+            b_feat = b_layer.GetNextFeature()
+
+        a_layer = None
+        b_layer = None
+        a_vector = None
+        b_vector = None

```

Comment:
Hey @emlys I think this function was removed in `release/3.9` but then maybe `git` incorrectly re-added it here?

In `release/3.9` we refactored some of our custom assertions into `natcap.invest.utils`, including this function.